apps/web/pages/[user]/book.tsx

```diff
@@ -3,6 +3,8 @@ import type { GetServerSidePropsContext } from "next";
 import type { LocationObject } from "@calcom/app-store/locations";
 import { privacyFilteredLocations } from "@calcom/app-store/locations";
 import { getAppFromSlug } from "@calcom/app-store/utils";
+import getBooking from "@calcom/features/bookings/lib/get-booking";
```

Comment:
More shared stuff. (And from here on I'll stop commenting this. If you see such movements, it's because the booker also needed this, and the functionality seemed like a shared thing to me)